src/containers/ResetPasswordPage.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+import React from 'react';
+import intl from 'react-intl-universal';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import './ResetPasswordPage.less';
+
+class ResetPasswordForm extends React.Component {
+  static propTypes = {
+    history: ReactRouterPropTypes.history.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      emailReset: '',
+      failedSubmit: false,
+    };
+    this.setEmailReset = this.setEmailReset.bind(this);
+    this.submitResetEmail = this.submitResetEmail.bind(this);
+  }
+
+  setEmailReset(event) {
+    this.setState({ emailReset: event.target.value });
+  }
+
+  submitResetEmail(event) {
+    event.preventDefault();
+    const { emailReset } = this.state;
+    if (!emailReset) {
+      this.setState({ failedSubmit: true });
+    }
+  }
+
+  cancel() {
+    const { history } = this.props;
+    history.push('/');
+  }
+
+  render() {
+    const {
+      emailReset, failedSubmit,
+    } = this.state;
+    return (
+      <div className="reset-password-form-component form-component container">
+        <div className="title">
+          {intl.get('reset-password')}
+        </div>
+        <div className="feedback-label feedback-container">
+          {failedSubmit ? intl.get('failed-to-save-message') : ''}
+        </div>
+        <form className="form-horizontal" onSubmit={this.submitResetEmail}>
+          <div className="form-group">
+            <div className="control-label" />
+            <div className="form-input">
+              {/* eslint-disable-next-line max-len */}
+              <p>
+                {intl.get('reset-email-txt')}
+              </p>
+              <input id="email_reset" name="EmailReset" className="form-control" type="email" value={emailReset} onChange={this.setOldPassword} />
```

Comment:
Where is `setOldPassword `? Without it, this form is not editable. The value is always set to an empty string.